examples/harmonic/python_driver/init.pdb

```diff
@@ -0,0 +1,4 @@
+TITLE cell{atomic_unit} positions{atomic_unit}
```

Comment:
xyz here? see pdb issue newly issued, no need to make a new one.

Reply:
It is done.